src/lib/converter/nodes/block.ts

```diff
@@ -112,8 +112,18 @@ export class BlockConverter extends ConverterNodeComponent<ts.SourceFile|ts.Bloc
 
         for (const symbol of context.checker.getExportsOfModule(moduleSymbol)) {
             const resolved = context.resolveAliasedSymbol(symbol);
-            for (const declaration of resolved.declarations) {
-                this.owner.convertNode(context, declaration);
+
+            const declarationsProcess = () => {
+                for (const declaration of resolved.declarations) {
+                    this.owner.convertNode(context, declaration);
+                }
+            };
+
+            if (symbol.getName() !== resolved.getName()) {
+                const reflection = createReferenceReflection(context, symbol, resolved);
+                context.withScope(reflection, declarationsProcess);
```

Comment:
This is kind of weird, conceptually a reference shouldn't have children, it should point to something that has children... Not necessarily wrong, but unexpected. It think might be better to just set the name of the converted reflection.